### ISSUE TITLE
Raspberry Pi displaying black

### ISSUE THREAD

**Issue by CodexDread** [rnayabed/streampi_client#2]
when running the client software on my raspberry pi 3 its displaying as a black screen, but all the buttons I set up still work and send commands through to the server side. It wasn't black during the entire time I was setting it and while using it the other day but today I got it to display once but had to make some changes and then it went right back to just being a black screen.

**Comment by RaySmith1** [comment]
Until streapi_client 'wakes' the display, you should adjust the desktop behavior for sleep/screensaver, as an alternative, you might be able to use xset in the streampi_client startup script.

```bash
# Disable Screen Sleep/Blank
xset s off
xset -dpms
xset s noblank
```

**Comment by CodexDread** [comment]
Alright well I have it set in the auto start after reboot mode so how do I prevent that because the program takes over before I can do anything.

**Comment by RaySmith1** [comment]
Hmm, the current installer places itself under /etc/rc.local for autostart.  You'd have to find a way to prevent it from starting.  The StreamPi Client, when running, doesn't allow TTY switching.  So you won't be able to switch to a TTY console to make changes.

I ran into the same issues you are describing.  I found the use of /etc/rc.local to launch the client was poor design.  A systemctl unit file would be best.   I used nodm and created a custom X session startup script.  When nodm is disabled, no GUI runs at all.   While not perfect, I think it's better than what is offered currently.  As a tip, while working on my StreamPi Client hardware, I enabled SSH as a fail-safe.   I know that doesn't resolve your current issue but I took time to document the build I used.  You might find it useful.  

https://github.com/RaySmith1/streampi_client/blob/StreamPi_Client_Build/StreamPi-Client-Build.md

If you would prefer to use a desktop image, you can create a desktop shortcut and manually start the client when needed.

```bash
HOME=~
cat <<-EOF > ${HOME}/Desktop/StreamPi.desktop
[Desktop Entry]
Type=Application
Terminal=false
Name=StreamPi
Exec=${HOME}/StreamPi/jre/bin/java -Dcom.sun.javafx.isEmbedded=true -Dcom.sun.javafx.touch=true -Dcom.sun.javafx.virtualKeyboard=javafx -jar StreamPiClient-0.6.jar
Path=${HOME}/StreamPi
Icon=/usr/share/pixmaps/debian-logo.png

EOF
```

At this point, a quick rebuild might be in your best interest.

**Comment by rnayabed** [comment]
will be fixed in upcoming release

**Comment by rnayabed** [comment]
Will close after it has been fixed

**Comment by rnayabed** [comment]
fixed in 1.0
